src/common/settings.h

```diff
@@ -56,11 +56,11 @@ enum class PortraitLayoutOption : u32 {
 enum class StereoRenderOption : u32 {
     Off = 0,
     SideBySide = 1,
-    ReverseSideBySide = 2,
-    Anaglyph = 3,
-    Interlaced = 4,
-    ReverseInterlaced = 5,
-    CardboardVR = 6
+    SideBySideFull = 2,
+    Anaglyph = 4,
```

Comment:
There's an off-by-one error starting at Anaglyph here. Was this intentional and there's a missing option, or should Anaglyph still equal 3?

Reply:
Oh man it's been so long I'm not sure! I think anaglyph should still be 3 because I'm replacing the ReverseSideBySide with SideBySideFull

Reply:
Yeah, that's what I thought, but I wanted to check to be sure. Not a big deal and an easy fix.

Reply:
I think an early version of this code included both and shifted the numbers up and this is an accidental remainder. Good catch!